src/Pages/form/Components/ArrivalOnset.tsx

```diff
@@ -0,0 +1,408 @@
+import "date-fns";
+import React, { useState } from "react";
+import { createStyles, makeStyles, withStyles } from "@material-ui/core/styles";
+import {
+  MuiPickersUtilsProvider,
+  KeyboardTimePicker,
+  KeyboardDatePicker,
+} from "@material-ui/pickers";
+import { MaterialUiPickersDate } from "@material-ui/pickers/typings/date";
+import Box from "@material-ui/core/Box";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import ToggleButton from "@material-ui/lab/ToggleButton";
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+import DateFnsUtils from "@date-io/date-fns";
+import intervalToDuration from "date-fns/intervalToDuration";
+
+const useStyle = makeStyles(() =>
+  createStyles({
+    textTitle: {
+      marginBottom: "16px",
+    },
+    boxPicker: {
+      marginLeft: "32px",
+      marginBottom: "40px",
+    },
+    boxOnset: {
+      marginLeft: "32px",
+      marginBottom: "40px",
+    },
+    boxOnsetPicker: {
+      marginTop: "70px",
```

Comment:
Actually, I recommend setting margin-bottom more than it. So, please set margin bottom at above element instead.